src/errors.ts

```diff
@@ -52,6 +52,12 @@ export class PlayerNotFoundError extends CustomError {
   }
 }
 
+export class OutfitAliasNotFoundError extends CustomError {
+  constructor() {
+    super('There is no PlanetSide 2 outfit with this alias')
```

Comment:
While `alias` is the term used in code, the term displayed to users should be "tag".